nipype/algorithms/tests/test_auto_ErrorMap.py

```diff
@@ -1,35 +0,0 @@
-# AUTO-GENERATED by tools/checkspecs.py - DO NOT EDIT
```

Comment:
Again, no ```test_auto_ ``` should be modified in this PR. Please update to current master and we'll see if there is some deviation from master that caused these tests to be updated.